robottelo/constants/__init__.py

```diff
@@ -952,13 +945,6 @@
         'view_discovery_rules',
     ],
     'Domain': ['view_domains', 'create_domains', 'edit_domains', 'destroy_domains'],
-    'Environment': [
```

Comment:
The test must be failing for `Environment` on satellite provided by satLab because by default we using a satellite that has puppet integration disabled. There has been some discussions happening around whether we should snap_template 7.0 with or without puppet as some components are still rely on puppet and we had to enable puppet for those components ! @pondrejk @jameerpathan111 has more insights about it than anyone!

So until we make that decision can we just comment these out instead of removing completely ?

Reply:
Sure thing, I'll make these changes now.